halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/kubernetes/KubernetesDistributedService.java

```diff
@@ -373,14 +359,7 @@ default KubernetesContainerDescription buildContainer(String name, ServiceSettin
     readinessProbe.setHandler(handler);
     container.setReadinessProbe(readinessProbe);
 
-    /* TODO(lwander) this needs work
```

Comment:
You guys rock

Reply:
:-)

To be completely honest, we only moved your TODO into the applyCustomSize method. Because we didn't implement handling of the SMALL/MEDIUM/LARGE specifiers, we kept your placeholder in its new home, below.

Reply:
Heh I saw that after